warp10/src/main/java/io/warp10/continuum/sensision/SensisionConstants.java

```diff
@@ -1025,6 +1025,16 @@
    */
   public static final String SENSISION_CLASS_CONTINUUM_ESTIMATORS_CACHED_PER_APP = "warp.estimators.cached.perapp";
 
+  /**
+   * Number of times the estimators was reset
+   */
+  public static final String SENSISION_CLASS_CONTINUUM_ESTIMATORS_RESETS = "warp.estimators.resets";
```

Comment:
Can you rename this metric to warp.estimator.resets as it concerns a single estimator.